### PR TITLE
Chore/put api

### PR DESCRIPTION
**Pull request by Yerish26**

### CODE REVIEW

File: backend/src/main/java/com/llm_service/llm_service/service/LLMService.java [hunk 1]
```diff
@@ -16,7 +16,7 @@ public class LLMService {
 
     public LLMService() {
         modelParameters =
-                new ModelParameters().setModelFilePath("model" + File.separator + "llama-2-13b-chat.Q5_K_S.gguf");
+                new ModelParameters().setModelFilePath("model" + File.separator + "model/llama-2-13b-chat.Q5_K_S.gguf");
```
Comment (Mgrdich) [inline backend/src/main/java/com/llm_service/llm_service/service/LLMService.java]: this should be like the old one ?

File: backend/src/main/java/com/llm_service/llm_service/service/ConversationService.java [hunk 2]
```diff
@@ -84,8 +78,9 @@ public void deleteAll() {
         conversationPersistenceManager.deleteAll();
     }
 
-    public Conversation edit(UUID id) {
-        return Conversation.builder().build();
+    public Conversation editTitle(UUID id, String title) {
+        return conversationPersistenceManager.save(
```
Comment (Mgrdich) [inline backend/src/main/java/com/llm_service/llm_service/service/ConversationService.java:82]: i think `editTitle` should throw an error when a conversation does not exists , here it is just getting optional from the `getByID`

File: backend/src/main/java/com/llm_service/llm_service/service/ConversationService.java [hunk 3]
```diff
@@ -35,26 +34,21 @@ public Optional<Conversation> getByID(UUID id) {
...
 
     // TODO optimize this fetching mechanism
-    public List<Discussion> askLLMQuestion(UUID id, ConversationRequest conversationRequest)
-            throws ConversationNotFoundException {
-        Optional<Conversation> conversation = conversationPersistenceManager.findById(id);
-
-        if (conversation.isEmpty()) {
-            throw new ConversationNotFoundException(id);
-        }
+    public List<Discussion> askLLMQuestion(ConversationRequest conversationRequest, Conversation conversation) {
```
Comment (Mgrdich) [inline backend/src/main/java/com/llm_service/llm_service/service/ConversationService.java:37]: maybe we should rename this `ConversationRequest` -> `LLMQuestionRequest` that way the function does not have two parameters that seems the same but are not.

**Review by Mgrdich — CHANGES_REQUESTED** [review]

File: backend/src/main/java/com/llm_service/llm_service/service/ConversationService.java (hunk 2 above)
Comment (Yerish26) [reply backend/src/main/java/com/llm_service/llm_service/service/ConversationService.java:82]: it is handled in the controller

**Pull request closed without merging**